src/test/java/com/meilisearch/sdk/TestUtils.java

```diff
@@ -0,0 +1,53 @@
+package com.meilisearch.sdk;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.lang.reflect.Type;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+public class TestUtils {
+
+	String movies_data;
+	Movie[] movies = new Movie[10];
+	Gson gson = new Gson();
+
+	public TestUtils () {
+		this.movies = moviesStringToJson(moviesFileToString());
+		return;
+	}
+
+	public String moviesFileToString () {
+		String content = "";
+		String filePath = "src/test/java/com/meilisearch/sdk/movies.json";
+		try
+		{
+			content = new String ( Files.readAllBytes( Paths.get(filePath) ) );
+		} 
+		catch (IOException e) 
+		{
+			e.printStackTrace();
+		}
+		this.movies_data = content;
+		return content;
+	}
```

Comment:
please put the json file unter src/test/resources/movies.json
src/test/java should only contain java files

Also, please [load the File from classpath](https://github.com/niemannd/meilisearch-java/blob/master/meilisearch-core/src/test/java/integrationtests/AbstractIT.java#L45).
This way we can move around our source files without to worry about where the json is located ;)